themes/delphi/layouts/partials/footer/desktop.html

```diff
@@ -0,0 +1,40 @@
+<div class="uk-visible@s footer">
+    <table class="uk-table footer-table">
+        <thead>
+            <tr>
+                <th class="header">
```

Comment:
should we prefix css classnames by their area?, e.g. `footer-header`?

Reply:
Will do